services/provider/server/server.go

```diff
@@ -2080,6 +2094,24 @@ func (s *OCSProviderServer) appendNoobaaKubeResources(
 	return kubeResources, nil
 }
 
+func (s *OCSProviderServer) getStorageClassResourceVersion(ctx context.Context) ([]string, error) {
+	storageClassesInCluster := &storagev1.StorageClassList{}
+	if err := s.client.List(ctx, storageClassesInCluster, &client.MatchingLabelsSelector{
+		// not select storageclass with external labels
+		Selector: util.GetExternalClassesBlacklistSelector(),
```

Comment:
is the list deterministic, better sort on names?

Reply:
ack, got it. Yes it could be one of the reasons